sunbird_mcp/src/server.py

```
@@ -0,0 +1,384 @@
+                This function modifies the artifact_urls list in the parent scope.
+            """
+            try:
+                api_url = f"{settings.API_BASE_URL.rstrip('/')}{CONTENT_ENDPOINT}/{content_id}"
+                async with session.get(api_url) as response:
+                    if response.status == 200:
+                        data = await response.json()
+                        content = data.get("result", {}).get("content", {})
+                        # Define excluded MIME type from config
+                        excluded_mime = settings.EXCLUDED_MIME_TYPE
+                        if content.get("mimeType") != excluded_mime and content.get("mimeType") == settings.PDF_MIME_TYPE and content.get("streamingUrl"):
+                            artifact_urls.append(content.get("streamingUrl"))
```

Ruff (0.11.9)

Undefined name `artifact_urls`

(F821)